### PR TITLE
Update lint/test/build infrastructure

### PR DESCRIPTION
**Pull request by mbengit**

### CODE REVIEW

File: qpylib/json_qpylib.py [hunk 1]
```diff
@@ -54,7 +53,6 @@ def _extract_type_from_context(context):
     return None
 
 def get_jsonld_type(jsonld_type):
-    global JSONLD_TYPES
```
Comment (jthomperoo) [inline qpylib/json_qpylib.py:57]: Just a question about this - will this still work if it's not set up as a global variable? What if for example you were using it in Flask/GUnicorn with multiple threads? Not entirely sure how this works but have hit issues with Python globals before.

File: qpylib/log_qpylib.py [hunk 2]
```diff
@@ -63,7 +63,6 @@ def create_log(syslog_enabled=True):
     }
 
 def log(message, level):
-    global LOG_LEVEL_TO_FUNCTION
```
Comment (jthomperoo) [inline qpylib/log_qpylib.py:66]: Same as above re:globals.

**Review by jthomperoo — COMMENTED** [review]
Just two comments to clarify - other than that looks good to me :)

File: qpylib/log_qpylib.py (hunk 2 above)
Comment (mbengit) [reply qpylib/log_qpylib.py:66]: Same reply as above.

**Review by jthomperoo — APPROVED** [review]

**Pull request merged**